vertx-web/src/client/vertx-eventbus.js

```diff
@@ -83,7 +83,9 @@
     this.defaultHeaders = null;
 
     // default event handlers
-    this.onerror = console.error;
+    this.onerror = function (err) {
+      console.error(err);
```

Comment:
Just a question,

is console.error supported by all browsers ? (should we check for definition first ?)